src/controllers/booksController.js

```diff
@@ -0,0 +1,25 @@
+const { StatusCodes } = require("http-status-codes");
+const { asyncWrapper } = require("../middlewares/asyncWrapperMiddleware");
+const { getBooksInfoService, getBookDetailService } = require("../services/booksService");
+
+/* 도서 목록 조회 */
+const getBooksInfo = async (req, res) => {
+  let { category_id: categoryId, new: isNew, page, limit } = req.query;
+
+  const { data } = await getBooksInfoService(categoryId, isNew, page, limit);
```

Comment:
```suggestion
  const { data } = await bookService.getBooksInfo(categoryId, isNew, page, limit);
```
property처럼 접근해서 쓰는 건 어떨까요?